doc/manual/manual.bib

```diff
@@ -11939,3 +11939,10 @@ @software{fraters_menno_2020_3900603
   doi          = {10.5281/zenodo.3900603},
   url          = {https://doi.org/10.5281/zenodo.3900603}
 }
+
+@phdthesis{Quinquis2014,
+author = {Quinquis, M},
+school = {Charles University},
+title = {{A numerical study of subduction zone dynamics using linear viscous to thermo-mechanical model setups including (de)hydration processes}},
+year = {2014}
+}
```

Comment:
I think you can drop this now, given that #4125 is merged.

Reply:
#4125 added the reference to the citation bib that is used to create the publication list on the website. In manual.tex, manual.bib is used instead of this citation bib. Is the reference nevertheless available?

Reply:
Ah no, I just didn't pay attention.